examples/export-pdf.js

```diff
@@ -4,8 +4,6 @@ import WKT from '../src/ol/format/WKT.js';
 import {Tile as TileLayer, Vector as VectorLayer} from '../src/ol/layer.js';
 import {OSM, Vector as VectorSource} from '../src/ol/source.js';
 
-import {toJpeg} from 'html-to-image';
```

Comment:
Can we remove this as a dependency?

Reply:
No, it is still used in the export-map example.